### DISCUSSION TITLE
Expand more the integration page

### DISCUSSION THREAD

**Discussion by ludgie** [labwc/labwc#1441] [Ideas]
Could get information from:

- https://github.com/swaywm/sway/wiki/Useful-add-ons-for-sway
- https://github.com/riverwm/river/wiki/Recommended-Software

Willing to do this if main developers like the idea

**Comment by ahesford** [comment]
Stealing content from other sites doesn't really add value. A section of the integration page linking to those external lists would be sufficient.

**Reply by ludgie** [reply]
Got it, thank you

**Comment by johanmalm** [comment]
I have made a start on a `links` page. Will add these two to that one.
